core/templates/pages/exploration-player-page/layout-directives/learner-view-info.component.spec.ts

```diff
@@ -208,6 +213,130 @@ describe('Learner view info component', () => {
+      outline: 'Outline',
+      exploration_id: 'exp_id_1',
+      outline_is_finalized: false,
+      exp_summary_dict: {
+        title: 'Title',
+        status: 'private',
+        last_updated_msec: 1591296737470.528,
+        community_owned: false,
+        objective: 'Test Objective',
+        id: '44LKoKLlIbGe',
+        num_views: 0,
+        thumbnail_icon_url: '/subjects/Algebra.svg',
+        human_readable_contributors_summary: {},
+        language_code: 'en',
+        thumbnail_bg_color: '#cc4b00',
+        created_on_msec: 1591296635736.666,
+        ratings: {
+          1: 0,
+          2: 0,
+          3: 0,
+          4: 0,
+          5: 0,
+        },
+        tags: [],
+        activity_type: 'exploration',
+        category: 'Algebra',
+      },
+      completed: true,
+      thumbnail_bg_color: '#927117',
+      thumbnail_filename: 'filename',
+    };
+
+    var secondSampleReadOnlyStoryNodeBackendDict = {
+      id: 'node_2',
+      description: 'description',
+      title: 'Title 2',
+      prerequisite_skill_ids: [],
+      acquired_skill_ids: [],
+      destination_node_ids: ['node_3'],
+      outline: 'Outline',
+      exploration_id: 'test_id',
+      outline_is_finalized: false,
+      exp_summary_dict: {
+        title: 'Title',
+        status: 'private',
+        last_updated_msec: 1591296737470.528,
+        community_owned: false,
+        objective: 'Test Objective',
+        id: '44LKoKLlIbGe',
+        num_views: 0,
+        thumbnail_icon_url: '/subjects/Algebra.svg',
+        human_readable_contributors_summary: {},
+        language_code: 'en',
+        thumbnail_bg_color: '#cc4b00',
+        created_on_msec: 1591296635736.666,
+        ratings: {
+          1: 0,
+          2: 0,
+          3: 0,
+          4: 0,
+          5: 0,
+        },
+        tags: [],
+        activity_type: 'exploration',
+        category: 'Algebra',
+      },
+      completed: false,
+      thumbnail_bg_color: '#927117',
+      thumbnail_filename: 'filename',
+    };
+
+    spyOn(urlService, 'getTopicUrlFragmentFromLearnerUrl').and.returnValue(
+      'topic_url_fragment'
+    );
+    spyOn(urlService, 'getClassroomUrlFragmentFromLearnerUrl').and.returnValue(
+      'classroom_url_fragment'
+    );
+    spyOn(urlService, 'getStoryUrlFragmentFromLearnerUrl').and.returnValue(
+      'story_url_fragment'
+    );
+
+    spyOn(storyViewerBackendApiService, 'fetchStoryDataAsync').and.returnValue(
+      Promise.resolve(
+        StoryPlaythrough.createFromBackendDict({
+          story_id: 'story_id',
+          story_nodes: [
+            firstSampleReadOnlyStoryNodeBackendDict,
+            secondSampleReadOnlyStoryNodeBackendDict,
+          ],
+          story_title: 'story',
+          story_description: 'Story_Description',
+          topic_name: 'Topic 1',
+          meta_tag_content: 'Story meta tag content',
+        })
+      )
+    );
+
+    componentInstance.ngOnInit();
+
+    tick();
+    tick();
```

Comment:
why are two `tick()`s needed?

Reply:
While testing it seemed like two were needed, but I tested it just with one and it worked, so I will correct that.